src/scenes/objects/cardObject.js

```diff
@@ -85,7 +85,9 @@ export class CardObject {
 
             const interaction = () => {
                 //Only trigger interaction if the dialogue is done
-                if(this.parent_scene.dialogue.isDone()) {
+                let interactionReady = this.parent_scene.dialogue ? this.parent_scene.dialogue.isDone() : true;
+
+                if(interactionReady) {
```

Comment:
This seems like a really convoluted solution. How about something like:
```javascript
if(this.parent_scene.dialogue) {
    if(this.parent_scene.dialogue.isDone()) {
        //Code
    }
}

```

Reply:
Which is essentially the same thing but without introducing an intermediary variable